packages/wfuzz/PKGBUILD

```diff
@@ -10,7 +10,7 @@ url='https://github.com/xmendez/wfuzz'
 arch=('any')
 license=('GPL2')
 depends=('python' 'python-lxml' 'python-pycurl' 'python-chardet' 'python-six'
-         'python-future' 'python-wxpython')
+         'python-future' 'python-wxpython' 'python-pyparsing')
```

Comment:
It seems this dependency is listed and expected https://github.com/xmendez/wfuzz/blob/1b695ee9a87d66a7d7bf6cae70d60a33fae51541/setup.py#L34